src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs

```diff
@@ -41,10 +41,15 @@ public class AnsiConsoleTheme : ConsoleTheme
         public static AnsiConsoleTheme Literate { get; } = AnsiConsoleThemes.Literate;
 
         /// <summary>
-        /// A theme in the style of the original <i>Serilog.Sinks.Literate</i> using only standard 16 terminal colors that will work on light backgrounds.
+        /// A theme in the style of the original <i>Serilog.Sinks.Sixteen</i> using only standard 16 terminal colors that will work on light backgrounds.
         /// </summary>
         public static AnsiConsoleTheme Sixteen { get; } = AnsiConsoleThemes.Sixteen;
 
+        /// <summary>
+        /// A theme in the style of the original <i>Serilog.Sinks.Dracula</i>.
```

Comment:
This seems like a placeholder comment that should be closer to the description added to the README